HttpKernel/ControllerInjectorsWarmer.php

```diff
@@ -83,10 +83,15 @@ private function findControllerClasses()
             $dirs[] = $controllerDir;
         }
 
-        foreach (Finder::create()->name('*Controller.php')->in($dirs)->files() as $file) {
-            $filename = $file->getRealPath();
-            if (!in_array($filename, $this->blackListedControllerFiles)) {
-                require_once $filename;
+        // Combination of scanAllBundles/scanBundles can lead to empty dirs.
+        // Only search for controllers if we have at least one directory,
+        // otherwise the finder will throw an exception.
```

Comment:
Please use multiline syntax for multiline comments

Reply:
Are you sure? There is a comment a few lines below and I used the same style...

Reply:
Oh ok forget it then I guess (but WTF though?)